### PR TITLE
Specify `envFrom` as `array` instead of `object` to support `valueFrom`

### PR DESCRIPTION
**Pull request by SergK**
We might have values defined in custom secrets/configmaps This change enables this capability

Related issue #82

### CODE REVIEW

File: charts/dependency-track/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: dependency-track
-version: 0.8.0
+version: 0.8.1
```
Comment (nscuro) [inline charts/dependency-track/Chart.yaml:3]: This is a breaking change, not a bugfix. I don't think we need to bump the major version as we're still v0, but this should at least be a minor bump.

**Review by nscuro — CHANGES_REQUESTED** [review]
Thanks for the PR!

Please also apply the change to the `values.yaml` files used in CI: https://github.com/DependencyTrack/helm-charts/tree/main/charts/dependency-track/ci

**Comment by SergK** [comment]
@nscuro thanks, fixed

**Review by nscuro — APPROVED** [review]

File: charts/dependency-track/templates/frontend/deployment.yaml [hunk 2]
```diff
@@ -48,9 +48,8 @@ spec:
         env:
         - name: API_BASE_URL
           value: {{ .Values.frontend.apiBaseUrl | quote }}
-        {{- range $k, $v := .Values.frontend.extraEnv }}
-        - name: {{ $k }}
-          value: {{ $v | quote }}
+        {{- with .Values.apiServer.extraEnv }}
```
Comment (niriak) [inline charts/dependency-track/templates/frontend/deployment.yaml:51]: Maybe `frontend` instead of apiServer` ?

Comment (nscuro) [reply charts/dependency-track/templates/frontend/deployment.yaml:51]: Good spot @niriak, a fix is on its way! #87

**Pull request merged**